horace_core/sqw/@SQWDnDBase/save.m

```diff
@@ -12,8 +12,13 @@
 %   w        -- sqw or dnd object or array of such objects.
 % Optional:
 %  filename  -- Filename for output. If none given, then prompted for
-%               a filename to save. If w is array, cellarray of file names
-%               must be provided here, one filename per each input object.
+%               a filename to save. If w is array you should either:
+%          single string with "filename.sqw" (.sqw must be present) should
+%               be provided. Then numel(w) files will be wirtten with the names
+%               filename_1.sqw,filename_2.sqw ... filename_n.sqw where
+%               n==numel(w)
+%          n arguments with filenames must be provided, one filename per
```

Comment:
```suggestion
%          or:
                   n arguments with filenames must be provided, one filename for
```
need the or to go with the either. Use for rather than per as per only really works with one word after